docs/tutorial/mint-nftstorage-polygon.md

```diff
@@ -312,12 +314,18 @@ You can expect to see the following output:
 NFT minted to: 0x{YOUR_WALLET_ADDRESS}
 ````
 
+#### Verify your NFT
+
+To verify your NFT, first go to Metamask test account. If minting was successful, your MATIC balance should reduce.
+
+To visually confirm your NFT, visit [Opensea Testnets](https://testnets.opensea.io/account), connect your Metamask test account, authorize. Then you should see your freshly minted NFT displayed under your account.
+
 ## Conclusion
 
-In this tutorial, we learned how to mint a NFT end-to-end with Polygon and NFT.Storage. This technology combination results in proper decentralization and guarantees *scalability*, *durability*, and *immutability*.
+In this tutorial, we learned how to mint an NFT end-to-end with Polygon and NFT.Storage. This technology combination results in proper decentralization and guarantees *scalability*, *durability*, and *immutability*.
 
 We deployed a custom smart contract to mint our NFT specific to our needs. For this tutorial, we used a simple example based on the ERC-721 standard. However, you can also define complex logic that governs your NFT life cycle. For more complex use cases, the successor standard [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/) is a good place to start. OpenZeppelin, the library we use in our tutorial offers a [contracts wizard](https://docs.openzeppelin.com/contracts/4.x/wizard) that helps create NFT contracts.
 
-Successful minting can be seen as the start of the valuable phase of the NFT. The NFT can then be used to prove ownership and can be transferred to other users. Reasons to transfer a NFT might include a successful sale on one of the NFT marketplaces like [OpenSea](https://opensea.io), or a different type of event such as acquiring an item in a NFT based game. Exploring the rich possibilities for NFTs is definitely an exciting next step.
+Successful minting can be seen as the start of the valuable phase of the NFT. The NFT can then be used to prove ownership and can be transferred to other users. Reasons to transfer an NFT might include a successful sale on one of the NFT marketplaces like [OpenSea](https://opensea.io), or a different type of event such as acquiring an item in an NFT based game. Exploring the rich possibilities for NFTs is definitely an exciting next step.
 
 If you\'d like help building your NFT project with NFT.storage, we encourage you to join the `#nft-storage` channel on [Discord](https://discord.gg/Z4H6tdECb9) and [Slack](https://filecoinproject.slack.com/archives/C021JJRH26B).
```

Comment:
Unrelated to the changes here, but since we're on this file... I think we should use this URL: `filecoin.io/slack` (it redirects to a URL with an embedded invite token that we rotate when it expires). Otherwise, new users will get an error saying they need an invite. The downside is that it doesn't link you directly to the channel, but I think the tradeoff is worth it.

Reply:
okay, agreed.